### DISCUSSION TITLE
Save to non-volatile Memory

### DISCUSSION THREAD

**Discussion by Davidwedel** [semuconsulting/PyGPSClient#126] [Q&A] [answered]
Hello all! I've used pygpsclient on and off for a few months now. Can confirm it works better than ucenter! 
My question: How do I save to eeprom(or whatever), so I don't lose the configuration on power cycle? I can't seem to find that...
Thanks!

**Comment by semuadmin** [comment, marked as answer]
Open the UBX Configuration panel and select the second option from the Presets list at the bottom right - “CFG-CFG Save configuration to non-volatile memory”.

Note that not all devices have EEPROM as such - many will have battery-backed RAM (BBR) or flash memory instead. 

![image](https://github.com/semuconsulting/PyGPSClient/assets/28569967/33553e95-ddeb-4b51-a652-387e85795c9e)

**Comment by Davidwedel** [comment]
Oh Ok! For some reason, I didn't think it was saving earlier... However, it is now. (I was probably doing something wrong:). Thanks! Great program.
